### DISCUSSION TITLE
Add newer php versions (8.1,8.2)

### DISCUSSION THREAD

**Discussion by steveberrill** [mgtcommerce/mgt-dev#25] [Q&A]
How can i add support for php 8.1 and 8.2 in this?
![php_version_problem](https://user-images.githubusercontent.com/3091095/230770927-0cbc5b35-c507-4a76-90b5-3371ea4a61ed.jpg)

**Comment by HowardPengg** [comment]
try docker for magento2 it can provide php 8.1 Version  easy run for developer
